python/katana/local/_graph.pyx

```diff
@@ -237,7 +243,7 @@ cdef class GraphBase:
             length ``self.num_nodes()``. (Optional)
         :param kwargs: Properties to add. The values must be arrays or sequences of length ``self.num_nodes()``. (Optional)
         """
-        handle_result_void(self.underlying_property_graph().UpsertNodeProperties(GraphBase._convert_table(table, kwargs)))
+        handle_result_RDGAccessT(self.underlying_property_graph().UpsertNodeProperties(GraphBase._convert_table(table, kwargs)))
```

Comment:
If we are discarding the access object, doesn't that mean we will be making any transaction generated from Python unsafe?